app/models/host/base.rb

```diff
@@ -366,7 +375,8 @@ def render_template(template:, **params)
     end
 
     def uptime_seconds
-      self.uptime_fact&.value&.to_i
+      boot_time = self&.reported_data&.boot_time
+      boot_time.nil? ? Time.zone.now.to_i - boot_time.to_i : nil
```

Comment:
This is reversed, test failures are most likely related to that:
```suggestion
      boot_time.present? ? Time.zone.now.to_i - boot_time.to_i : nil
      # OR
      boot_time && Time.zone.now.to_i - boot_time.to_i
```

Reply:
yeah, I figured while fixing tests :-)